mllib/src/main/scala/org/apache/spark/ml/attribute/AttributeGroup.scala

```diff
@@ -165,6 +165,11 @@ class AttributeGroup private (
   /** Converts to a StructField. */
   def toStructField(): StructField = toStructField(Metadata.empty)
 
+  override def toString: String = {
```

Comment:
Is it required? Please keep this PR minimal. You can create another PR if `toString` is useful for AttributeGroup. (Btw, we can reuse the JSON serialization in `toString`.)